databricks/koala/structures.py

```diff
@@ -195,6 +195,10 @@ def getField(self, name):
                     "Field {} not found, possible values are {}".format(name, ", ".join(fnames)))
             return anchor_wrap(self, self._spark_getField(name))
 
+    # TODO: automate the process here
+    def alias(self, name):
```

Comment:
With this fix, I am not even sure we need to overwrite this function.

Reply:
I just removed for now, but if we have this, `col.alias("x").name` will return `x`.
Currently the original `Column` methods are not anchor the dataframe reference except for methods in `_wrap_operators`.
We might need to wrap them automatically as the same.